tests/run_tests.py

```diff
@@ -238,6 +238,13 @@ def _parse_command_line():
 
     _initialize_logging(options.log_to_console)
 
+    # If we have a custom test root, add its python folder, if it exists, so the user doesn't need
+    # to set it up themselves.
```

Comment:
the test root already gets written out when running tests right? If not, that could be potentially useful to output.